src/features/metadata-instance-editor/CascadePolicy.js

```diff
@@ -36,38 +37,48 @@ const CascadePolicy = ({
     ) : null;
 
     return canEdit ? (
-        <div className="metadata-cascade-editor">
-            <div className="metadata-cascade-enable">
-                <div>
-                    <FormattedMessage tagName="strong" {...messages.enableCascadePolicy} />
-                    {!isCustomMetadata && (
-                        <Toggle
-                            className={`metadata-cascade-toggle ${isCascadingEnabled ? 'cascade-on' : 'cascade-off'}`}
-                            isOn={isCascadingEnabled}
-                            label=""
-                            onChange={e => onCascadeToggle(e.target.checked)}
-                        />
-                    )}
-                </div>
-                {!isCustomMetadata ? (
+        <React.Fragment>
+            <div className="metadata-cascade-editor">
+                <div className="metadata-cascade-enable">
                     <div>
-                        <FormattedMessage {...messages.applyCascadePolicyText} />
-                        &nbsp;
-                        <Link className="cascade-policy-learnmore-link" href={COMMUNITY_LINK} target="_blank">
-                            <FormattedMessage {...messages.cascadePolicyLearnMore} />
-                        </Link>
-                    </div>
-                ) : (
-                    <div>
-                        <FormattedMessage {...messages.cannotApplyCascadePolicyText} />
+                        <FormattedMessage tagName="strong" {...messages.enableCascadePolicy} />
+                        {!isCustomMetadata && (
+                            <Toggle
+                                className={`metadata-cascade-toggle ${
+                                    isCascadingEnabled ? 'cascade-on' : 'cascade-off'
+                                }`}
+                                isOn={isCascadingEnabled}
+                                label=""
```

Comment:
Hmm, off topic, we probably should add `hideLabel` for Toggle, like we have it for TextInput / TextArea.
And label should still be sent for accessibility reasons.